setup.py

```diff
@@ -12,16 +12,20 @@ def requirements_from_file(filename):
             if line.strip() and not line.strip().startswith('--')]
 
 
-def read(fname, URL):
+def read(fname, URL, URLImage):
     """Read the content of a file."""
     readme = open(path.join(path.dirname(__file__), fname)).read()
     if hasattr(readme, 'decode'):
         # In Python 3, turn bytes into str.
         readme = readme.decode('utf8')
     # turn relative links into absolute ones
+
```

Comment:
This new newline is useless and counter productive: it takes the comment above apart from the corresponding lines of code.